tests/e2e/editor-selector.spec.js

```diff
@@ -0,0 +1,38 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('Test the creation of a new Post', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
+  await page.goto('./');
+  await login(page, context);
+
+  // Creating and navigating to new post
+  await page.goto('./wp-admin/post-new.php');
+
+  // On dev instance, need to close modal so test can continue
+  const closeButton = page.locator('.components-modal__header button');
```

Comment:
Does this mean that the page creation modal no longer shows on local? Because it should 🤔 Could this be due to the WordPress version maybe?

Reply:
I have no idea, but on my local it does not show up, also the flow is a little different, the work around I had earlier made things slow so I refactored it.

Reply:
If you use the VSCode extension and run the test in debug mode, it opens a webpage and shows you the tests running visually, that way you can see the difference.